element_lab/lab.py

```diff
@@ -18,18 +18,41 @@ def activate(schema_name, create_schema=True, create_tables=True):
     )
 
 
+@schema
+class Organization(dj.Imported):
+    definition = """# Top-level list of all organizations involved in any of the projects
+    organization_name          : VARCHAR(256)                # full organization name
```

Comment:
Please use a shorter type for the primary key.  Use an abbreviated name for primary key with a full name as a secondary attribute.